saleor/webhook/serializers.py

```diff
@@ -87,7 +87,9 @@ def untaxed_price_amount(price: TaxedMoney) -> Decimal:
     return [
         {
             **_get_checkout_line_payload_data(line_info),
-            "unit_amount": untaxed_price_amount(line_info.line.unit_price),
+            "unit_amount": untaxed_price_amount(
+                line_info.line.total_price / line_info.line.quantity
```

Comment:
Not sure if `untaxed_price_amount` already does this, but keep in mind that you should quantize_price here

Reply:
![image](https://user-images.githubusercontent.com/43955230/168085614-9a08b264-55e0-435b-a88c-0d443f00147b.png)
`quantize_price` is used inside `untaxed_price_amount`.
Do you want to extract it from this function?

Reply:
IMO we can extract it